src/notebooks.scss

```diff
@@ -0,0 +1,162 @@
+// "Preview" notebook styles, which are here (versus in the component style file) because the HTML is dynamically generated
+// see https://stackoverflow.com/questions/56265483/angular-how-to-style-dynamically-generated-html
+
+.notebook {
+  display: grid;
+  grid-template-columns: fit-content(7em) auto;
+  grid-template-rows: auto;
+  grid-auto-flow: row;
+  padding: 1em 1.5em 1em 1em;
+  border: solid 1px #cfd1d7;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.1);
+  border-radius: 4px;
+  line-height: 1.5;
+  font-size: 12px;
+
+  h1 {
+    font-size: 24px;
```

Comment:
The front end style guide says we should use rem instead of pixels

Reply:
True, although I'm guessing that the intent was related to sizing dialogs and divs, rather than fonts.  In this case, our notebook styles adjust the global styles for some elements, the font sizes of which are also defined in `px`, so it's more likely that our notebook will look ok and exist harmoniously with its surroundings if we define its font sizes in `px`, too.  (Fun fact: about 20% of the `font-size:` in our styles are set with `rem` units.)